main/sdr.go

```diff
@@ -244,9 +236,12 @@ func (u *UAT) sdrConfig() (err error) {
 	} else {
 		log.Printf("\tResetBuffer Successful\n")
 	}
+
 	//---------- Get/Set Freq Correction ----------
 	freqCorr := u.dev.GetFreqCorrection()
 	log.Printf("\tGetFreqCorrection: %d\n", freqCorr)
+
+	u.ppm = getPPM(u.serial)
```

Comment:
Suggest to add 
`log.Printf("===== UAT Device Serial: %s PPM %d =====\n", u.serial, u.ppm)`